api/views/workflow.py

```diff
@@ -234,3 +238,102 @@ def get(self, request, *args, **kwargs):
+
+    ## Submit Data in SHARE v1 Format
+    Please note that this endpoint is to ease the transition from SHARE v1 to SHARE v2 and sources
+    are encouraged to transition to submitting metadata in the SHARE v2 format.
+
+    Submitting data through the normalizeddata endpoint is strongly preferred as support for
+    the v1 format will not be continued.
+
+    v1 Format
+
+        For the full format please see https://github.com/erinspace/shareregistration/blob/master/push_endpoint/schemas.py
+
+        Required Fields: [
+            "title",
+            "contributors",
+            "uris",
+            "providerUpdatedDateTime"
+        ],
+
+    Create
+
+        Method:        POST
+        Body (JSON): {
+                        {
+                            "publisher":{
+                                "name": <publisher name>,
+                                "uri": <publisher uri>
+                            },
+                            "description": <description>,
+                            "contributors":[
+                                {
+                                    "name":<contributor name>,
+                                    "email": <email>,
+                                    "sameAs": <uri>
+                                },
+                                {
+                                    "name":<contributor name>
+                                }
+                            ],
+                            "title": <title>,
+                            "tags":[
+                                <tag>,
+                                <tag>
+                            ],
+                            "languages":[
+                                <language>
+                            ],
+                            "providerUpdatedDateTime": <time submitted>,
+                            "uris": {
+                                "canonicalUri": <uri>,
+                                "providerUris":[
+                                    <uri>
+                                ]
+                            }
+                        }
+
+                    }
+        Success:       200 OK
+    """
+    permission_classes = [ReadOnlyOrTokenHasScopeOrIsAuthenticated, ]
+    serializer_class = NormalizedDataSerializer
+
+    def get_queryset(self):
+        return NormalizedData.objects.all()
+
+    def post(self, request, *args, **kwargs):
+
+        prelim_data = request.data
+        app_label = request.user.username
+
+        # store raw data, assuming you can only submit one at a time
+        raw = None
+        with transaction.atomic():
+            try:
+                doc_id = prelim_data['uris']['canonicalUri']
+            except KeyError:
+                return Response({'errors': 'Canonical URI is required.', 'data': prelim_data}, status=status.HTTP_400_BAD_REQUEST)
+            except:
```

Comment:
If it is not an expected error we should just let django handle it and send back a 500 to the user.